libs/shared/ui/src/lib/components/header/header.tsx

```diff
@@ -18,13 +18,13 @@ export function Header(props: HeaderProps) {
   const { title, icon, buttons, actions, iconClassName = '' } = props
 
   return (
-    <div className="flex h-32 border-b border-neutral-200 items-center justify-between bg-white rounded-t p-5 shrink-0">
+    <div className="flex border-b border-neutral-200 items-center justify-between bg-white rounded-t px-5 py-8 shrink-0">
       <div className="flex gap-4 ml-2 items-center">
         {icon && <Icon name={icon} className={`w-16 ${iconClassName}`} />}
```

Comment:
```suggestion
{icon && (
  <div className="flex items-center justify-center w-16 h-16 rounded-full border border-neutral-200">
      <Icon name={icon} className={`w-10 ${iconClassName}`} />
  </div>
)}
```

I think we could add it to be more iso with design and the header information are less compact, wdyt ?